src/_layouts/default.html

```diff
@@ -0,0 +1,50 @@
+<!doctype html>
+<html lang="en">
+  <head>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+      <title>Home</title>
```

Comment:
Note that this `title` isn't informative or dynamic. It can be replaced with the site name or, if you want it to be dynamic, it could follow the pattern in my [11ty demo](https://glitch.com/edit/#!/mica-11ty-demo-done).